README.md

```diff
@@ -125,3 +125,26 @@ Or if you are using Ninja:
 [Ninja]: https://ninja-build.org
 [SPIRV-Headers]: https://github.com/KhronosGroup/SPIRV-Headers
 [SPIRV-Tools]: https://github.com/KhronosGroup/SPIRV-Tools
+
+## Builtin Library
+
+Clspv includes an LLVM IR library (cmake/clspv--.bc) containing implementations
+of some OpenCL builtin functions. This file is distributed under the LLVM
+license (included in LICENSE) as it is generated from the LLVM sub-project
+[libclc](libclc.llvm.org). Refer to the source for the relevant copyrights.
+
+### Rebuilding the Library
+
+#### Required Tools
+
+You will need a pre-built version of [LLVM](github.com/llvm/llvm-project) for
```

Comment:
Does the version of this LLVM build matter? (e.g. not any newer than the version used by Clspv?)

Reply:
Given we try to stay on top-of-trunk it shouldn't really matter. Generally, this library will not need to be regenerated often.